server/src/domain/services/mail-service.js

```diff
@@ -34,7 +34,33 @@ function sendFeedbackEmail(form) {
   return mailJet.sendEmail(options);
 }
 
+function sendJobsChangedEmail(form) {
```

Comment:
Je pense que ça peut marcher... MAIS !

Après ce que j'aime bien c'est que ça donne une homogénéité entre les différentes méthodes (toutes prennent un objet model `form`).

Par ailleurs, dans ce cas, il faudrait reconstuire un objet passé en paramètre de la ligne `const template = jobsChangedEmailTemplate.compile(form);` qui le re-déconstruit à l'intérieur.

Je suis pour laisser en l'état si tu veux bien. Sinon je me range à ton avis.